optaplanner-quickstarts/quarkus-facility-location/pom.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.acme</groupId>
+  <artifactId>optaplanner-quarkus-facility-location-quickstart</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+
+  <properties>
+    <version.org.optaplanner>8.0.0-SNAPSHOT</version.org.optaplanner>
+    <version.io.quarkus>1.6.0.Final</version.io.quarkus>
+
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <failsafe-plugin.version>2.22.1</failsafe-plugin.version>
+    <surefire-plugin.version>2.22.2</surefire-plugin.version>
+    <docker-plugin.version>0.28.0</docker-plugin.version>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.quarkus</groupId>
+        <!-- Alternatively, use <artifactId>quarkus-universe-bom</artifactId>
+         which includes both quarkus-bom and optaplanner-bom. -->
+        <artifactId>quarkus-bom</artifactId>
+        <version>${version.io.quarkus}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.optaplanner</groupId>
+        <artifactId>optaplanner-bom</artifactId>
+        <version>${version.org.optaplanner}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.optaplanner</groupId>
+      <artifactId>optaplanner-quarkus</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.optaplanner</groupId>
+      <artifactId>optaplanner-quarkus-jackson</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy-jackson</artifactId>
+    </dependency>
```

Comment:
soft suggestion on order:
define quarkus-resteasy* before optaplanner-quarkus* because in a demo when building the app, the funcionality order typically goes: first rest (resteasy), then database (hibernate, not applicable here), then AI (optaplanner).

Reply:
I started with optaplanner, then I added REST.